src/utils/database-common/src/testing/fake_database_transaction_manager.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use internal_error::InternalError;
+use sqlx::SqlitePool;
+
+use crate::{DatabaseTransactionManager, TransactionRef};
+
+///////////////////////////////////////////////////////////////////////////////
+
+pub struct FakeDatabaseTransactionManager {
```

Comment:
Why not to use `dill::Component` ?

Reply:
This requires registering `SqlitePool` as well -- I thought it best to leave knowledge of `SqlitePool` only at the `FakeDatabaseTransactionManager` level, and not take it higher to the `FakeDatabasePlugin` level

Reply:
I suggest we leave it as it is